modules/lfs/transferadapter.go

```diff
@@ -37,6 +37,8 @@ func (a *BasicTransferAdapter) Download(ctx context.Context, l *Link) (io.ReadCl
 	if err != nil {
 		return nil, err
 	}
+	req.Header.Del("Accept")
```

Comment:
Instead of deleting, can you find the place where the header is originally set and see why it sets it in first place?

Reply:
Cool, I will try to understand more about the purpose behind it.